CMakeLists.txt

```diff
@@ -153,8 +149,23 @@ if (MFEM_USE_CUDA)
        # nvcc
        set(MFEM_CUDA_COMPILER_IS_NVCC ON)
        set(CUDA_FLAGS "${CUDA_FLAGS} --expt-extended-lambda --expt-relaxed-constexpr")
+       if("all" STREQUAL "${CMAKE_CUDA_ARCHITECTURES}" or "native" STREQUAL "${CMAKE_CUDA_ARCHITECTURES}" or "all-major" strequal "${CMAKE_CUDA_ARCHITECTURES}")
```

Comment:
For consistency, replace `strequal` -> `STREQUAL`.

Also, please try to split long lines to fit in 80 columns.